26-register_registration/2451-creator-adds-a-contributor.feature

```diff
@@ -0,0 +1,56 @@
+        Given Creator navigates to Contributors tab
+        And the Registration is a "<RegistrationType>"
+        When they click "Add Contributor"
+        Then they see the "Add Contributor" Dialog
+        And they see a dropdown with Contributor Types "<ContributorTypes>"
+        And they see a "Close" Button
+        And they see a "Create new Contributor" Button
+        And they see a "Add me as Contributor" Button
+        And they see a disabled "Add" Button
+        Examples:
+            | RegistrationType          | ContributorTypes                                                                     |
+            | JournalArticle            | Contact person, Editor, Related person, Researcher, Rights holder, Supervisor, Other |
+            | JournalShortCommunication | Contact person, Editor, Related person, Researcher, Rights holder, Supervisor, Other |
+            | FeatureArticle            | Contact person, Editor, Related person, Researcher, Rights holder, Supervisor, Other |
+            | JournalLetter             | Contact person, Editor, Related person, Researcher, Rights holder, Supervisor, Other |
+            | JournalReview             | Contact person, Editor, Related person, Researcher, Rights holder, Supervisor, Other |
+            | JournalLeader             | Contact person, Editor, Related person, Researcher, Rights holder, Supervisor, Other |
+            | JournalCorrigendum        | Contact person, Editor, Related person, Researcher, Rights holder, Supervisor, Other |
+            | BookMonograph             | Contact person, Editor, Related person, Researcher, Rights holder, Supervisor, Other |
+            | BookAnthology             | Contact person, Related person, Researcher, Rights holder, Supervisor, Other         |
+            | ReportResearch            | Contact person, Editor, Related person, Researcher, Rights holder, Supervisor, Other |
+            | ReportPolicy              | Contact person, Editor, Related person, Researcher, Rights holder, Supervisor, Other |
+            | ReportWorkingPaper        | Contact person, Editor, Related person, Researcher, Rights holder, Supervisor, Other |
+            | ReportBasic               | Contact person, Editor, Related person, Researcher, Rights holder, Supervisor, Other |
+            | DegreeBachelor            | Contact person, Editor, Related person, Researcher, Rights holder, Other             |
+            | DegreeMaster              | Contact person, Editor, Related person, Researcher, Rights holder, Other             |
+            | DegreePhd                 | Contact person, Editor, Related person, Researcher, Rights holder, Other             |
+            | OtherStudentWork          | Contact person, Editor, Related person, Researcher, Rights holder, Other             |
+            | ChapterArticle            | Contact person, Editor, Related person, Researcher, Rights holder, Supervisor, Other |
```

Comment:
kan disse bli alfabetisk sortert, vær så snill? :) :) :)

Reply:
Er det ikke litt greit at de er gruppert på samme måte som de er i appen også? `FeatureArticle` er feks en subtype av `Journal`, men subtypen har ikke samme prefiks som resten. samme gjelder `OtherStudentWork` (som er `Degree`). hvis vi lister opp alfabetisk mister vi de sammenhengene :/

Reply:
Denne lista blir jo veldig lang etterhvert da. Det skal jo komme mange flere typer. Grupperinger er fint så lenge det er oversiktlig synes jeg, men dette ser ikke oversiktlig ut for meg...

Reply:
okei, tror ikke det blir seende oversiktlig ut uansett for min del, så da blir det alfabetisering :p

Reply:
Haha, er jo ikke sikkert at mitt forslag er det beste for alle da, så noen flere burde mene noe slik at det blir flertallet som bestemmer, men 😅